client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -312,12 +317,14 @@ function getFallbackDestination( {
 		if ( isJetpackNotAtomic && purchasedProduct ) {
 			debug( 'the site is jetpack and bought a jetpack product', siteSlug, purchasedProduct );
 
+			const adminPath = adminPageRedirect || 'admin.php?page=jetpack#/recommendations';
```

Comment:
Could you also edit the tests in [this file](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js) to cover this modification?